pkgs/applications/audio/strawberry/default.nix

```diff
@@ -62,6 +63,8 @@ mkDerivation rec {
     taglib
     qtbase
     qtx11extras
+  ] ++ lib.optionals (lib.versions.major qtbase.version == "5") [
```

Comment:
I wondering if we should introduce `isQt5` and `isQt6` helpers like what we have for Python as this check (or a variation of it) is most likely going to be littered all over the place.